apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts

```diff
@@ -221,6 +221,9 @@ export class SubscriptionsPlansComponent implements OnInit, OnDestroy {
         }
     }
 
+    Navigate(elem: HTMLElement) {
```

Comment:
Method name must be camelCase
Do use proper param names in method **element**
Add JS doc and return type

Reply:
fixed